### DISCUSSION TITLE
Tremaine website

### DISCUSSION THREAD

**Discussion by Tremainegreen** [community/community#23956] [Other Feature Feedback, Questions, & Ideas]
<p>Hello eveyone,</p>
<p>I am new to Github. I’ve been following tutorials etc on how to set up pages. Now, i’m trying to move my codepen HTML and CSS over here. HTML is working, but CSS is not being applied.</p>
<p>I’m obviously doing something wrong. But I can’t figure it out.</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/Tremainegreen/TremaineGreenIT" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:400/400;"><img src="https://avatars.githubusercontent.com/u/68483975?s=400&amp;amp;v=4" class="thumbnail" width="400" height="400"></div>

<h3><a href="https://github.com/Tremainegreen/TremaineGreenIT" target="_blank" rel="noopener nofollow ugc">Tremainegreen/TremaineGreenIT</a></h3>

  <p><span class="label1">Contribute to Tremainegreen/TremaineGreenIT development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Thank you in advance for any pointers.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The problem is that your <code>index.html</code> is referencing <code>css/styles.css</code>, but the directory in your repository is called <code>Css/</code>:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Tremainegreen/TremaineGreenIT/blob/5e22617f39c8a0b02c23ef185d5b687c9de7ec26/index.html#L12" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Tremainegreen/TremaineGreenIT/blob/5e22617f39c8a0b02c23ef185d5b687c9de7ec26/index.html#L12" target="_blank" rel="noopener">Tremainegreen/TremaineGreenIT/blob/5e22617f39c8a0b02c23ef185d5b687c9de7ec26/index.html#L12</a></h4>
<pre class="onebox"><code class="lang-html"><ol class="start lines" start="2" style="counter-reset: li-counter 1 ;">
<li>&lt;html lang="en"&gt;
</li>
<li>&lt;head&gt;
</li>
<li>  &lt;meta charset="UTF-8"&gt;
</li>
<li>  &lt;meta name="viewport" content="width=device-width, initial-scale=1.0"&gt;
</li>
<li>  &lt;meta http-equiv="X-UA-Compatible" content="ie=edge"&gt;
</li>
<li> 
</li>
<li> 
</li>
<li>  
</li>
<li>  &lt;title&gt;TremaineGreenIT&lt;/title&gt;
</li>
<li>  
</li>
<li class="selected">  &lt;link rel="stylesheet" href="css/styles.css"&gt;
</li>
<li>  &lt;link rel="icon" href="favicon.ico"&gt;
</li>
<li>  &lt;link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500&amp;family=Sacramento&amp;display=swap" rel="stylesheet"&gt;
</li>
<li>
</li>
<li>&lt;/head&gt;
</li>
<li>&lt;body&gt;
</li>
<li>&lt;div class="top-container"&gt;
</li>
<li>  &lt;img class="top-cloud"src="images/cloud.png" alt="Cloud"&gt;
</li>
<li> &lt;img src="images/TG_logo_gold_2_1024x1024.png" alt="T"&gt;
</li>
<li>  
</li>
<li>  
</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>You need to either rename the directory or change the <code>href</code> path. If it works on your machine that probably means you’re using Windows, almost everywhere else paths are case-sensitive.</p>

**Comment by tqpcharlie** [comment]
<p>I’ve opened a PR at <a href="https://github.com/Tremainegreen/TremaineGreenIT/pull/1" class="inline-onebox" rel="noopener nofollow ugc">adds capital "C" by Zachinquarantine · Pull Request #1 · Tremainegreen/TremaineGreenIT · GitHub</a> that should fix it…</p>

**Comment by Tremainegreen** [comment]
<p>oooo Kudos thank you</p>
